pkg/segment/segment.go

```diff
@@ -215,11 +216,12 @@ func GetUserIdentity(telemetryFilePath string) (string, error) {
 	}
 
 	// check if the id is a valid uuid, if not, nil is returned
```

Comment:
Also, please update this comment above, which is no longer accurate.

Reply:
Fixed in https://github.com/redhat-developer/odo/pull/7173/commits/e4bb039e7a7d89b58f9d12865ec164cf49d571cf.